crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -938,6 +938,7 @@ pub enum ConvertFlakeRefError {
 }
 
 #[cfg(test)]
+
```

Comment:
I don't think there's supposed to be a blank line here.

Reply:
```suggestion
```